config/v1/types_cluster_version.go

```diff
@@ -95,6 +95,13 @@ type ClusterVersionStatus struct {
 	// +required
 	Desired Release `json:"desired"`
 
+	// architecture indicates whether the cluster is a homogeneous or
```

Comment:
What transitions are allowed for this value?

Reply:
all transitions are theoretically allowed.  multi -> amd64 and amd64 -> multi should be fairly straightforward.  To do amd64 -> s390x, you'd need to do some fairly exciting dances, likely passing through multi in between.

Reply:
are we saying that if there is a transition from adm64->s390x, it will involve a middle stage of first moving to multi and then to the other single arch? 

i'm wondering if single -> multi should be the only allowed transition.